### ISSUE TITLE
Samsung account sign-in plugin

### ISSUE THREAD

**Issue by swift-kim** [flutter-tizen/plugins#490]
I was trying to implement a Tizen version of [google_sign_in](https://pub.dev/packages/google_sign_in) or [sign_in_with_apple](https://pub.dev/packages/sign_in_with_apple) which can be used to authorize a user on TV devices, but realized that there's not enough public API open to 3rd-party developers (even those who have partner-level certificates).

According to the `Tizen.TV.Service.Sso.dll` file in the [Tizen.NET.TV](https://www.nuget.org/packages/Tizen.NET.TV) package, most of its public methods require the platform-level privilege (`http://developer.samsung.com/privilege/sso.platform`) which is only granted to internal (VD) developers. The only information that can be obtained with a partner-level certificates is:

- Login status (`Sso.GetLoginStatus`)
- User's GUID (`Sso.GetGuid`)
- User's UID (`Sso.GetLoginUid`)
- User's service subscription list (`Sso.GetCpAccountMappingList`)

If you have a partner-level certificate and want to know how to get the above information in C# or Dart code, please let me know.

The following APIs are only available for internal developers:

- `Sso.GetLoginInfo`
- `Sso.RequestAccessToken`
- `Sso.SubscribeLoginState`
- `Sso.RequestAuthcode`
- `Sso.RequestUserAuthcode`
- `Sso.RequestDeviceAuthcode`
- `Sso.RequestAccountList`

In short, the Tizen.NET.TV API doesn't meet the OAuth standard and cannot be wrapped into a Dart API gracefully.
